pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+line-length = 88
```

Comment:
88 is the default, but we can swap to whatever. 88 is what the SynapsePythonClient is using